api/bash/gitline

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+DOMAIN="https://api.github.com"
+source ~/projects/github_bash_project/.privateVariables.sh
+
+# Assigning Arguments
+whichMethod=$1
+username="$2"
+label_state_or_repo_name=$3
+is_private=$4
+
+function fetchIssuesWhereIAmAssignedWithLabel {
+  response=$(curl --silent ${DOMAIN}/search/issues?q=+label:${label_state_or_repo_name}+assignee:${username}+state:open&sort=created&order=asc)
```

Comment:
@brntbeer: I believe @PatrickBM started down that path with a $GH_TOKEN environment variable (see the [plumbing](https://github.com/github/platform-samples/blob/add-shell-scripts/api/bash/plumbing)). I'm not really sure what's a better approach.